cg/apps/demultiplex/sample_sheet/sample_models.py

```diff
@@ -85,6 +87,9 @@ def _update_barcode_mismatches_1(
     ) -> None:
         """Assign zero to barcode_mismatches_1 if the hamming distance between self.index
         and the index1 of any sample in the lane is below the minimum threshold."""
+        if not self.index:
+            self.barcode_mismatches_1 = "na"
```

Comment:
Is "na" an allowed value? From the documentation online it looks like it only accepts 0, 1 or 2. https://support.illumina.com/help/BS_App_BCL_Convert/v110_1000000130493/Content/SW/Apps/SampleSheetSettings_swBCL_swBS_appBCL.htm

Reply:
Resolved in person